docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -194,6 +194,8 @@ const columns: GridColDef[] = [
 
 {{"demo": "pages/components/data-grid/columns/RenderCellGrid.js", "bg": "inline"}}
 
+**Note**: It is recommended to also set a `valueFormatter` providing a representation for the value to be used when [exporting](/components/data-grid/export/) the data.
```

Comment:
It would be better to point directly to the section.

```suggestion
**Note**: It is recommended to also set a `valueFormatter` providing a representation for the value to be used when [exporting](/components/data-grid/export/#export-custom-rendered-cells) the data.
```